packages/foam-vscode/static/graphs/default/graph.js

```diff
@@ -27,7 +27,7 @@ const style = {
 const sizeScale = d3
   .scaleLinear()
   .domain([0, 30])
-  .range([1, 3])
+  .range([0.5, 2])
```

Comment:
Nitpick for the future, changes like these add noise to the original PR 
I also suffer from this but I think we can all each other get better